source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +235,63 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', async () => {
+      if (process.platform === 'darwin') {
```

Comment:
Can the tray creation be put in a function? Because in #33 the tray must be created after the preference is set to true. The tray creation function will be called externally from `source/win-preferences/preferences.vue`.